bin/deploy.sh

```diff
@@ -56,6 +56,13 @@ chk_config()
     fi
     echo "=> k8s Context is: \"${context}\""
 
+    if [ "${context}" = "minikube" ]; then
+        echo "=> Minikube deployment detected.  Installing tiller..."
+        helm init --service-account default --upgrade
```

Comment:
I'm not crazy about this being in the deploy script

It should be part of the minikube bringup